src/aruku/walking/node/walking_node.cpp

```diff
@@ -34,69 +35,52 @@
 namespace aruku
 {
 
-std::string WalkingNode::get_node_prefix()
-{
-  return "walking";
-}
+std::string WalkingNode::get_node_prefix() { return "walking"; }
 
-std::string WalkingNode::set_walking_topic()
-{
-  return get_node_prefix() + "/set_walking";
-}
+std::string WalkingNode::set_walking_topic() { return get_node_prefix() + "/set_walking"; }
 
-std::string WalkingNode::status_topic()
-{
-  return get_node_prefix() + "/status";
-}
+std::string WalkingNode::status_topic() { return get_node_prefix() + "/status"; }
 
-std::string WalkingNode::set_odometry_topic()
-{
-  return get_node_prefix() + "/set_odometry";
-}
+std::string WalkingNode::set_odometry_topic() { return get_node_prefix() + "/set_odometry"; }
 
 WalkingNode::WalkingNode(
   rclcpp::Node::SharedPtr node, std::shared_ptr<WalkingManager> walking_manager)
-: walking_manager(walking_manager)
+: walking_manager(walking_manager), disable_publish_joint(false)
 {
   set_walking_subscriber = node->create_subscription<SetWalking>(
-    set_walking_topic(), 10,
-    [this](const SetWalking::SharedPtr message) {
+    set_walking_topic(), 10, [this](const SetWalking::SharedPtr message) {
```

Comment:
Ditto.

Reply:
this also used on locomotion_node too mas

Reply:
Also replace that in the `locomotion_node`.